src/styles/main.scss

```diff
@@ -0,0 +1,37 @@
+@import "./blocks/minute.scss";
+@import "./blocks/second.scss";
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  display: flex;
+  padding: 70px;
+  justify-content: center;
+  align-items: center;
+  background-color: #cecece;
+}
+
+.watch {
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-image: url("../img/kisspng-clock-face-template-clock-position-clip-art-clock-without-hands-5b445014f15ae4.9351921415312036049886.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 400px;
+  height: 400px;
+  border: 1px solid black;
+  border-radius: 50%;
+
+  &__circle {
+    position: absolute;
+    display: inline-block;
+    width: 20px;
+    height: 20px;
+    background-color: red;
+    border-radius: 50%;
+    
+  }
```

Comment:
[BEM] - create a separate file per each BEM block styles that have the same name as the block